[
  {
    "_id": "62b0e8beda1c4fc55064d49d",
    "site_id": "62abd8b53653340a3be6cdeb",
    "queued_time": "2022-06-20T16:38:06.159000",
    "start_time": "2022-06-20T16:38:06.668000",
    "end_time": "2022-06-20T16:38:13.612000",
    "last_active": "2022-06-20T16:38:06.701000",
    "status": "CANCELED",
    "documents_found": 3,
    "new_documents_found": 0,
    "worker_id": "9cd10042-6e1f-45d8-a592-95bae5fc41d9",
    "error_message": "Traceback (most recent call last):\n  File \"/Users/andrewhorn/workspace/Apollo/backend/scrapeworker/main.py\", line 145, in worker_fn\n    try:\n  File \"/Users/andrewhorn/workspace/Apollo/backend/scrapeworker/scrape_worker.py\", line 258, in run_scrape\n    await self.wait_for_completion_or_cancel(downloads)\n  File \"/Users/andrewhorn/workspace/Apollo/backend/scrapeworker/scrape_worker.py\", line 190, in wait_for_completion_or_cancel\n    await asyncio.gather(self.watch_for_cancel(tasks), *tasks)\n  File \"/Users/andrewhorn/workspace/Apollo/backend/scrapeworker/scrape_worker.py\", line 184, in watch_for_cancel\n    raise CanceledTaskException(\"Task was canceled.\")\nbackend.scrapeworker.scrape_worker.CanceledTaskException: Task was canceled.\n",
    "links_found": 284
  },
  {
    "_id": "62b0c7bff47f13df91d2852f",
    "site_id": "62abd8b53653340a3be6cdeb",
    "queued_time": "2022-06-20T14:17:19.185000",
    "start_time": "2022-06-20T14:17:22.497000",
    "end_time": "2022-06-20T14:17:22.594000",
    "last_active": "2022-06-20T14:17:22.594000",
    "status": "FAILED",
    "documents_found": 0,
    "new_documents_found": 0,
    "worker_id": "aac5ae05-ec4d-41cf-ab84-54a3b90ea3b7",
    "error_message": "Traceback (most recent call last):\n  File \"/Users/andrewhorn/workspace/Apollo/backend/scrapeworker/main.py\", line 145, in worker_fn\n    raise Exception()\nException\n",
    "links_found": 0
  }
]
